### PR TITLE
docker fixes

### PR DESCRIPTION
**Pull request by Lxkas**

### CODE REVIEW

File: dockerfile [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
-# Copy the rest of the application files to the container
+# ---- Build ----
+FROM dependencies AS build
 COPY . .
-
-# Build the production version of the application
 RUN npm run build
 
-# Expose the port that your Next.js application will run on (by default Next.js uses port 3000)
+# ---- Release with Distroless ----
```
Comment (chinathaip) [inline dockerfile:15]: oh I thought we would simply add postcss to prod dep instead of this

Comment (Lxkas) [reply dockerfile:15]: nah, this fixes the issue too, and should prevent this type of thing from cropping up again

Comment (chinathaip) [reply dockerfile:15]: aight cool then

**Review by chinathaip — APPROVED** [review]

File: .dockerignore [hunk 2]
```diff
@@ -1,21 +1,39 @@
...
+
+# Ignore git files
+.git/
 .gitignore
 
-# Ignore build and output files
-/.next
-/out
+# Ignore CI/CD config files (if any)
+.gitlab-ci.yml
```
Comment (chinathaip) [inline .dockerignore:26]: nit: wtf XD

Comment (Lxkas) [reply .dockerignore:26]: chatgpt XD

**Pull request merged**